app/src/main/java/com/intervaltimer/google/activities/MainActivity.java

```diff
@@ -142,51 +141,46 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
         }
 
         Intent receivedIntent = getIntent();
-        Log.i(LONG_INTENT_TAG, "Intent in onCreate() triggered");
+        Log.i(LOG_INTENT_TAG, "Intent in onCreate() triggered");
```

Comment:
I did not check the remaining code, but how do they handle logging at the moment?